### DISCUSSION TITLE
[Bug?]: Incorrect k9s colors

### DISCUSSION THREAD

**Discussion by ventsislav-georgiev** [ghostty-org/ghostty#3611] [Q&A] [answered]
Using the following custom theme:
```
palette = 0=#212430
palette = 1=#df625b
palette = 2=#5dc875
palette = 3=#e4cf67
palette = 4=#4689cc
palette = 5=#c24480
palette = 6=#519f98
palette = 7=#ede8d7
palette = 8=#576383
palette = 9=#d48387
palette = 10=#77b88a
palette = 11=#f4b994
palette = 12=#68b0f9
palette = 13=#b688dd
palette = 14=#65cbce
palette = 15=#ced2e7
background = #000000
foreground = #ced2e7
cursor-color = #4783ef
selection-background = #212430
selection-foreground = #96a1a1
```

Ghostty looks like this:
<img width="995" alt="image" src="https://github.com/user-attachments/assets/635591a4-e76b-4f5e-900f-b889b2a32328" />

Opening yazi looks like this:
<img width="769" alt="image" src="https://github.com/user-attachments/assets/f8943692-4741-4dd1-bed5-29da5ec04691" />

k9s looks all bland (no skins used):
<img width="1132" alt="image" src="https://github.com/user-attachments/assets/6712e510-b7e8-4592-afb4-764c26a82965" />

here is same in **iTerm2**:
<img width="1081" alt="image" src="https://github.com/user-attachments/assets/35748282-124c-4382-87db-f60805a858ad" />

**Comment by mitchellh** [comment, marked as answer]
k9s previously ignored our terminfo (bad). Can you try with `TERM=xterm-256color` and see what happens?

**Reply by rhodes-b** [reply]
#2125 this would also be slightly involved?

**Reply by ventsislav-georgiev** [reply]
Thanks, setting `TERM=xterm-256color` fixes it.

**Reply by mitchellh** [reply]
Note that's not a proper long term fix, you (or me or someone) should report a bug to k9s that they're not using terminfo correctly.

**Reply by ventsislav-georgiev** [reply]
I'm not familiar with the specifics of terminfo usage in order to make a proper bug report.
